modules/express/src/fetchEncryptedPrivKeys.ts

```diff
@@ -0,0 +1,46 @@
+import { writeFile } from 'fs';
+import { BitGo } from 'bitgo';
+
+// TODO: set env to 'test' or 'prod'
+const bg = new BitGo({ env: 'test' });
+// TODO: set your access token here
+const accessToken = '';
+// TODO: set your coin type and wallet ids here e.g.
+// const walletIds = [
+//   ['tbtc', '61f039aad587c2000145c687373e0fa1'],
+//   ['tgeth', '61fb21819c14dd000755f8de3a18e461']
+// ];
+const walletIds = [];
```

Comment:
Maybe make this an object with each coin name as a key, and an array of wallet ids as the value
```suggestion
const walletIds = {
//   tbtc: ['61f039aad587c2000145c687373e0fa1'],
//   tgeth: ['61fb21819c14dd000755f8de3a18e461'],
};
```